tests/functional/test_manage.py

```diff
@@ -14,7 +15,8 @@ def test_manage_make_migrations(test_project):
 def test_manage_migrate(test_project):
     test_project.run_cmd('testproject/manage.py db create .models')
     test_project.run_cmd('testproject/manage.py db upgrade')
-    test_project.run_cmd('sqlite3 testproject.db .tables | grep user')
+    res = sqlite3.connect('testproject.db').cursor().execute("SELECT count(*) FROM sqlite_master WHERE type='table' AND name='user'; ").fetchall()
```

Comment:
Quite a huge line, isn't it?

Reply:
Yes, it is. That was my first reaction too, but I chose to leave it like this because it's pretty easy to follow.

Do you think I should split it and make it more explicit?

Reply:
I'd split it into multiple lines. Reasons include: (1) editor/linter warnings, (2) if we ever decide to run autoformatting tools such as `black`, it will do it anyway.  If you really want this to be a one-liner, you could extract the query into a variable.

Reply:
done